pkg/utils/deployer.go

```diff
@@ -605,6 +605,59 @@ func ApplyResourceWithRetry(ctx context.Context, dynamicClient dynamic.Interface
 	return lastError
 }
 
+func ApplyResource(ctx context.Context, dynamicClient dynamic.Interface, restMapper meta.RESTMapper,
```

Comment:
`ApplyResourceWithRetry` is used only once. You can modify in-place, instead of having another function `ApplyResource` doing the same thing.